src/app/hooks/useGlean.ts

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useEffect } from "react";
+import Glean from "@mozilla/glean/web";
+import * as appEvents from "../../telemetry/generated/appEvents";
+
+// Custom hook that initializes Glean and returns the Glean objects required
+// to record data.
+export const useGlean = () => {
+  // Initialize Glean only on the first render
+  // of our custom hook.
+  useEffect(() => {
+    Glean.initialize("monitor-testing", true, {
```

Comment:
```suggestion
    Glean.initialize("monitor.frontend", true, {
```

I believe we should change this before merging the PR. I think `monitor.frontend` might be an appropriate choice (it follows the same naming pattern as [FxA](https://dictionary.telemetry.mozilla.org/apps/accounts_frontend?itemType=app_ids&page=1)). @rhelmer thoughts?